src/Bundle/spec/Sender/Adapter/SymfonyMailerAdapterSpec.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\MailerBundle\Sender\Adapter;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Component\Mailer\Event\EmailSendEvent;
+use Sylius\Component\Mailer\Model\EmailInterface;
+use Sylius\Component\Mailer\Renderer\RenderedEmail;
+use Sylius\Component\Mailer\Sender\Adapter\AbstractAdapter;
+use Sylius\Component\Mailer\SyliusMailerEvents;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\RawMessage;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+final class SymfonyMailerAdapterSpec extends ObjectBehavior
```

Comment:
I'm missing two cases here:
1. Sending an email with attachments
2. Crash during the email is sent (`TransportExceptionInterface` is thrown)